extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudStreamExtractor.java

```diff
@@ -55,14 +57,14 @@ public String getName() {
 
     @Nonnull
     @Override
-    public String getTextualUploadDate() {
-        return track.getString("created_at");
+    public String getTextualUploadDate() throws ParsingException {
+        return track.getString("created_at").replace("T"," ").replace("Z", "");
```

Comment:
Why is this change needed? Also, I don't think those additional imports are needed here.

Reply:
For some reason SoundCloud api-v2 returns upload date in format "2016-07-31 18:18:07" instead of "2016/07/31T18:18:07Z+0000"

Reply:
For some reason SoundCloud api-v2 returns upload date in format "2016-07-31 18:18:07" instead of "2016/07/31T18:18:07Z+0000", so even though `getTextualUploadDate` should return the scraped date as a fallback, I converted it to a human readable one since it takes no effort and works reliably.

Reply:
Will remove the imports though